### ISSUE TITLE
attr_encrypted

### ISSUE THREAD

**Issue by didlix** [KatanaCode/genderize#4]
Does this work with attr_encrypted fields? Our database has the field encrypted_gender, which is then decrypted and mapped to gender.

**Comment by Bodacious** [comment]
I don't think it will @didlix  - The genderize method defines getters and setters that read and write directly from and to the column

https://github.com/KatanaCode/genderize/blob/master/lib/genderize.rb#L47-L52
